builtin/providers/google/resource_sql_database_instance.go

```diff
@@ -1109,3 +1211,20 @@ func resourceSqlDatabaseInstanceDelete(d *schema.ResourceData, meta interface{})
 
 	return nil
 }
+
+func validateDay(v interface{}, k string) (ws []string, errors []error) {
+	return validateNumericRange(v, k, 1, 7)
+}
+
+func validateHour(v interface{}, k string) (ws []string, errors []error) {
+	return validateNumericRange(v, k, 0, 23)
```

Comment:
Why don't we just call the `validateNumericRange func` directly? Seems weird having a wrapper